app/celery/letters_pdf_tasks.py

```diff
@@ -1,424 +0,0 @@
-        )
-
-
-def _move_invalid_letter_and_update_status(
-    notification,
-    filename,
-    scan_pdf_object,
-):
-    try:
-        move_scan_to_invalid_pdf_bucket(filename)
-        scan_pdf_object.delete()
-
-        update_letter_pdf_status(
-            reference=notification.reference, status=NOTIFICATION_VALIDATION_FAILED, billable_units=0
-        )
-    except BotoClientError:
-        current_app.logger.exception('Error when moving letter with id %s to invalid PDF bucket', notification.id)
-        update_notification_status_by_id(
-            notification.id, NOTIFICATION_TECHNICAL_FAILURE, status_reason='Internal server error'
-        )
-
-
-def _upload_pdf_to_test_or_live_pdf_bucket(
-    pdf_data,
-    filename,
-    is_test_letter,
-):
-    target_bucket_config = 'TEST_LETTERS_BUCKET_NAME' if is_test_letter else 'LETTERS_PDF_BUCKET_NAME'
-    target_bucket_name = current_app.config[target_bucket_config]
-    target_filename = get_folder_name(datetime.utcnow(), is_test_letter) + filename
-
-    s3upload(
-        filedata=pdf_data,
-        region=current_app.config['AWS_REGION'],
-        bucket_name=target_bucket_name,
-        file_location=target_filename,
-    )
-
-
-def _sanitise_precompiled_pdf(
-    self,
-    notification,
-    precompiled_pdf,
-):
-    try:
-        response = requests_post(
-            '{}/precompiled/sanitise'.format(current_app.config['TEMPLATE_PREVIEW_API_HOST']),
-            data=precompiled_pdf,
-            headers={
-                'Authorization': 'Token {}'.format(current_app.config['TEMPLATE_PREVIEW_API_KEY']),
-                'Service-ID': str(notification.service_id),
-                'Notification-ID': str(notification.id),
-            },
-            timeout=(3.05, 1),
-        )
-        response.raise_for_status()
-        return response
-    except RequestException as e:
-        current_app.logger.exception(e)
-
-        if e.response is not None and e.response.status_code == 400:
-            message = f'sanitise_precompiled_pdf validation error for notification: {notification.id}.'
-            if 'message' in response.json():
-                message += ' ' + response.json()['message']
-
-            current_app.logger.error(message)
-            return None
-
-        try:
-            current_app.logger.error('sanitise_precompiled_pdf failed for notification: %s', notification.id)
-            self.retry(queue=QueueNames.RETRY)
-        except MaxRetriesExceededError:
-            current_app.logger.error(
-                'RETRY FAILED: sanitise_precompiled_pdf failed for notification %s', notification.id
-            )
-            notification.status = NOTIFICATION_TECHNICAL_FAILURE
-            dao_update_notification(notification)
-            raise
-
-
-@notify_celery.task(name='process-virus-scan-failed')
-def process_virus_scan_failed(filename):
-    move_failed_pdf(filename, ScanErrorType.FAILURE)
```

Comment:
I believe ` move_failed_pdf` can be removed from the `app.letters.utils.py` file as it's only called there and in testing